Automattic/simplenote-electron
# Sync note after restoration to update the state

### Fix
Fixes #1772

After restoring a note the editor was stuck o the selected revision. This PR
clears that revision from the flux state. Additionally, that restoration of a
note was not causing a sync to occur.  This triggers a note sync that updates
everything and cements the restored version as the current version.

### Test
1. Open a note with history
2. Open revision panel
3. Select a version
4. Restore a version
5. Refresh
6. Is note set to revision?
7. Redo steps above, without refresh
8. Select another note
9. Select note that you accepted revision on
10. Is the note still on the revision?

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated with:
